website/content/api-docs/auth/aws.mdx

```diff
@@ -1378,9 +1378,9 @@ $ curl \
     http://127.0.0.1:8200/v1/auth/aws/tidy/identity-accesslist
 ```
 
-## Deprecations effective in Vault 1.7
+## Deprecations effective in OpenBao 1.7
 
-Vault 1.7 introduced new URLs for a number of AWS Auth APIs. The previous
+OpenBao 1.7 introduced new URLs for a number of AWS Auth APIs. The previous
```

Comment:
@joewxboy Something worth discussing is if we want a fresh start without legacy versions....

@JanMa strictly since we've not ironed out versioning yet, I think this should be reverted back to Vault as we're unlikely to retroactively attempt to rebuild and maintain previous versions as well.